### ISSUE TITLE
Homepage: Download Catalog button does not work

### ISSUE THREAD

**Issue by jtuckertw** [Tallwave/AEF-PIC#19]
Jeff to link files Dewan provided here : https://drive.google.com/open?id=13Vsjzpy4OcFL0ZcLtXXciWpPRS8Y41MY

Instead of a catalog we will be creating a page of product sheet lists.  These are the first that Dewan has provided.

**Comment by victorjaquez** [comment]
hi @jtuckertw I need the correct pdf to upload to the backend, can you provide this please?

**Comment by jtuckertw** [comment]
@kylielujan Please work with Dewan to determine what is needed for the catalog - either removal of button or to be supplied with a  digital catalog.

**Comment by aguillen1** [comment]
@jtuckertw Have they provided this catalog for us to upload to the backend yet?

**Comment by jtuckertw** [comment]
@aguillen1 During the meeting a few weeks back they wanted to go the library route:  https://docs.google.com/document/d/1uSlLyz-dszyvsecVX4KuVMKOalBnJraNmbjZlfNRC44/edit

Sent Dewan's email to top of your inbox.  Let me know if you have questions.

**Comment by victorjaquez** [comment]
hey @jtuckertw i don't see the link to any pdf's in the doc above? just notes...

**Comment by jtuckertw** [comment]
@victorjaquez 
- [ ] Relabel buttons to Downloads instead of Download Catalog
- [ ] Create page that contains download links 
- [ ] DOwnloads should link to these PDFs: 

https://drive.google.com/drive/folders/13Vsjzpy4OcFL0ZcLtXXciWpPRS8Y41MY

**Comment by victorjaquez** [comment]
@jtuckertw waiting on instruction design wise on this

**Comment by jtuckertw** [comment]
Spoke with @cmoosman1 who said using the News Template might be a possibility, but the problem with that solution is that all the other news content comes with it.  I think the news template could work if the news could be mutually exclusive of the Download content.

If the above solution is a dead-end, is it possible to produce a simple page with Header/footer and a list of upload links in between?  Think: A duplicate of the Contact Screen, just no form, and just a simple bulleted list of  text and links.

**Comment by victorjaquez** [comment]
@jtuckertw @cmoosman1 @jthiel16 not sure how to move forward on this

**Comment by jthiel16** [comment]
@jtuckertw @victorjaquez @cmoosman1 I like the idea of creating a mutually exclusive "Downloads" page, using the News template (as long as we leave the news items off the new page). 

The button on the home page could say "See Downloads" and then the Downloads page would have a card for each brochure. Victor, can we do that somewhat easily?

**Comment by jtuckertw** [comment]
@victorjaquez @cmoosman1 Let's regroup on this if possible on Monday.  We need a very basic landing page for the downloads to be downloaded from and another similar basic page for non-press release content that can be added (ex. "Christmas Parade news" item)

**Comment by victorjaquez** [comment]
@jtuckertw i'm working on the downloads page, can you walk over when you get a chance so we can speak about the non-press release content

**Comment by victorjaquez** [comment]
@jtuckertw downloads page is done

**Comment by victorjaquez** [comment]
@jtuckertw can we create a separate ticket for the new non press release page above please

**Comment by jtuckertw** [comment]
@victorjaquez Done!  #65

**Comment by jtuckertw** [comment]
This looks awesome -  THANK YOU SO MUCH!  @victorjaquez @cmoosman1
